src/Application/Filter/AdmissionRequirementTypeFilter.php

```diff
@@ -10,6 +10,7 @@
 final class AdmissionRequirementTypeFilter implements Filter
 {
     public const FREE = 'frei';
+    public const FREE_WITH_RESTRICTION = 'frei-mit-einschrankung';
```

Comment:
Let's write the "ä" umlaut in "Einschränkung" as "ae" just like in the next line in "eingeschränkt".
```suggestion
    public const FREE_WITH_RESTRICTION = 'frei-mit-einschraenkung';
```
In fact, we are forced to do this because it is used in the tax query, right? And the term slug uses "ae", too.